wp-content/themes/vf-wp-sis/partials/head.php

```diff
@@ -39,5 +39,10 @@
     }
 }
 
+$articleMigrated = get_field('art_migrated_from_drupal');
+if($articleMigrated === true){
+    $cssClass .= ' sis-article-legacy';
```

Comment:
I've also changed this to `sis-article-legacy`, I assume you don't mind?

Reply:
Yes, no problem :-)

Reply:
Please merge it if you are ready. I could deploy it later on dev.beta if you want.